src/stac_api_benchmark/query.py

```diff
@@ -110,8 +110,8 @@ async def search(
             f"sortby = {sortby}, datetime = {datetime}, "
             f"filter = {json.dumps(cql2_filter) if cql2_filter else ''}"
         )
+        t_start = perf_counter()
```

Comment:
rework the perf counter so it still works when called directly rather than through gather (this still needs a bit of work, but it at least doesn't cause the entire run to crash if there's an exception running on of the benchmarks)